### ISSUE TITLE
version label value comes empty for kube_state_metrics_build_info 

### ISSUE THREAD

**Issue by eminaktas** [kubernetes/kube-state-metrics#1670]
**What happened**:

kube-state-metrics (v2.3.0) shows `version` metric label empty for `kube_state_metrics_build_info`.

```bash
# HELP kube_state_metrics_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which kube_state_metrics was built.
# TYPE kube_state_metrics_build_info gauge
kube_state_metrics_build_info{..omitted..,version=""} 1
```

**What you expected to happen**:

Like in the previous versions we should be able to see values of `version` label.

```bash
# HELP kube_state_metrics_build_info A metric with a constant '1' value labeled by version, revision, branch, and goversion from which kube_state_metrics was built.
# TYPE kube_state_metrics_build_info gauge
kube_state_metrics_build_info{..omitted..,version="2.3.0"} 1
```

**How to reproduce it (as minimally and precisely as possible)**:

```
helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
helm repo update
helm upgrade --install prometheus-operator prometheus-community/kube-prometheus-stack --set kube-state-metrics.image.tag=v2.3.0
```

**Anything else we need to know?**:

**Environment**:
- kube-state-metrics version: v2.3.0
- Kubernetes version (use `kubectl version`): v1.20.7
- Cloud provider or hardware configuration: 
- Other info:

cc @necatican @yasintahaerol @f9n @developer-guy @dentrax

**Comment by dgrisonnet** [comment]
I couldn't reproduce your issue with the manifests provided in this repository, could you perhaps try?

Also, it is worth noting that this information is loaded into the binary so if you are using the official image `k8s.gcr.io/kube-state-metrics/kube-state-metrics:v2.3.0`, you should be able to see the version in the `kube_state_metrics_build_info` metric. Could you perhaps verify that it is the case in your setup?

**Comment by eminaktas** [comment]
@dgrisonnet thanks for the quick response.

Actually, we are using the [bitnami container image](https://hub.docker.com/layers/bitnami/kube-state-metrics/2.3.0-debian-10-r26/images/sha256-60a3852ecb8a84494021a42cdc9e07fc16a9d2546207aad520dfc0b7c3348b66?context=explore) I don't why but the older version is broken somehow. I checked the newest image it works perfectly. Also, I checked on the official image, it works.
